lms/djangoapps/verify_student/models.py

```diff
@@ -351,11 +351,22 @@ class PhotoVerification(IDVerificationAttempt):
     # capturing it so that we can later query for the common problems.
     error_code = models.CharField(blank=True, max_length=50)
 
+    # Mark if attempt is going through an A/B experiment
+    experiment_name = models.CharField(null=True, max_length=50)
```

Comment:
Will `experiment_name` differ between the two experiences in a single experiment?

Reply:
@bseverino and I thought it would be a good idea to have a separate string for each experiment or variation being run. That way if we have one experiment with several variations within it, we will be able to differentiate it based on this field